bin/publish

```diff
@@ -36,11 +32,11 @@ for image_name in "${IMAGES[@]}"; do
 
     for tag in "${TAGS[@]}" $(gen_versions $VERSION); do
       echo "Tagging and pushing $REGISTRY/$image_name:$tag"
-      docker tag "$image_name:$TAG" "$REGISTRY/$image_name:$tag"
+      docker tag conjur-authn-k8s-client:dev "$REGISTRY/$image_name:$tag"
```

Comment:
Can you extract this `conjur-authn-k8s-client:dev` to a variable though? it would be good that we make it obvious that both images are sourced from the same source docker image which isn't too apparent here unless you go through it extremely carefully.